Makefile

```diff
@@ -166,8 +168,12 @@ endif
              -e GOOGLE_APPLICATION_CREDENTIALS=./spacemesh.json \
              -e CLIENT_DOCKER_IMAGE="spacemeshos/$(DOCKER_IMAGE_REPO):$(BRANCH)" \
              -it go-spacemesh-python pytest -s sync/test_sync.py --tc-file=sync/config.yaml --tc-format=yaml
-
-
+else
```

Comment:
this can be refactored without duplication of the entire command,
use ifndef TEST -> delimiter='::' else delimiter=''
and then the last line should look like this
-it go-spacemesh-python pytest -s test_bs.py$(Delimiter)$(TEST) --tc-file=config.yaml --tc-format=yaml

Reply:
ok.